build/form.js

```diff
@@ -0,0 +1,108 @@
+var inquirer, _;
+
+_ = require('lodash');
+
+inquirer = require('inquirer');
+
+
+/**
+ * @summary Translate a form description to inquirer questions
+ * @function
+ * @private
+ *
+ * @param {Object[]} form - form description
+ * @returns {Object[]} inquirer questions
+ *
+ * @todo This is a work in progress and only necessary input types are supported.
+ *
+ * @example
+ * questions = visuals.form.parse [
+ *		label: 'Processor'
+ *		name: 'processorType'
+ *		type: 'select'
+ *		values: [ 'Z7010', 'Z7020' ]
+ *	,
+ *		label: 'Coprocessor cores'
+ *		name: 'coprocessorCore'
+ *		type: 'select'
+ *		values: [ '16', '64' ]
+ *	,
+ *		label: 'HDMI'
+ *		name: 'hdmi'
+ *		type: 'checkbox'
+ *		value: 1
+ * ]
+ */
+
+exports.parse = function(form) {
+  return _.map(form, function(option) {
+    var result;
+    result = {
+      message: option.label,
+      name: option.name
+    };
+    if (!_.isEmpty(option["if"])) {
+      result.when = function(answers) {
+        var conditions;
+        conditions = _.map(option["if"], function(condition) {
+          var key, value;
+          for (key in condition) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.